### DISCUSSION TITLE
FR: more complicated names for folder notes

### DISCUSSION THREAD

**Discussion by miscoined** [LostPaul/obsidian-folder-notes#228] [Ideas]
It'd be nice if the folder note name setting allowed for more complicated names.

My use case is that I use the [Johnny.Decimal](https://johnnydecimal.com/) folder organisation method, which means that my folder names are prefixed with numbers. So I might have a folder named eg `11.13 Twice Cooked Pork` which contains my notes about that recipe.

Ideally, I'd like to be able to use a Folder Note here for the 'main' page for whatever goes in that folder, but have the actual note name without the decimals, so that I can still write eg `I made [[Twice Cooked Pork]] today` and have it link to the folder note. Right now, it would have to be `I made [[11.13 Twice Cooked Pork|Twice Cooked Pork]] today` instead which I find extremely cumbersome to do everywhere in my vault.

One option would be to add a setting which allows users to provide a regex and a replacement string which is applied to the folder name, rather than just a static string.

**Comment by gypradhan** [comment]
+1 for this feature. I have a much simpler use case. My folder names are in plural and I'd like my folder note names to be in singular. Eg. Folder note named **Project** in the **Projects** folder
